Makefile

```diff
@@ -113,12 +118,28 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/crio
+	rm -f bin/crio.cross.*
 	$(MAKE) -C conmon clean
 	$(MAKE) -C pause clean
 	rm -f test/bin2img/bin2img
 	rm -f test/copyimg/copyimg
 	rm -f test/checkseccomp/checkseccomp
 
+# the approach here, rather than this target depending on the build targets
+# directly, is such that each target should try to build regardless if it
+# fails. And return a non-zero exit if _any_ target fails.
+local-cross:
```

Comment:
I still think the [`--keep-going` approach][1] is cleaner than this, but I'm fine addressing that in follow-up work if you don't want to pick it up here.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1506#discussion_r192862408

Reply:
i'd be game for that if there were a decorator for that target to not require the user to just happen to know that flag.

Reply:
oh i see a way.